src/app/personside/visittkort-v2/FeilendeSystemAdvarsel.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import styled from 'styled-components/macro';
+import { ReactNode } from 'react';
+import theme from '../../../styles/personOversiktTheme';
+
+const BeskrivelseDiv = styled.div`
```

Comment:
[TIPS] tror vi bør prøve å unngå å skrive `Div` i variabelnavnet. For om vi finner ut at det heller burde være en `span`, så "må" vi egentlig endre variabelnavnet også.